addons/linecompass/stringtable.xml

```diff
@@ -5,74 +5,92 @@
             </Key>
             <Key ID="STR_dui_linecompass_icon_outline">
                 <English>Icon Outline</English>
                 <French>Contour des icônes</French>
+                <German>Icon Umriss</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_icon_color">
                 <English>Default Icon Color</English>
                 <French>Couleur de l'icône par défaut</French>
+                <German>Standardfarbe des Icons</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_icon_color_desc">
                 <English>Default color for icons without fire team assignment</English>
                 <French>Couleur par défaut des icônes sans affectation à une équipe</French>
+                <German>Standardfarbe für Icons ohne Zuweisung zu einem Feuerteam</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_waypoint_color">
                 <English>Default Waypoint Color</English>
                 <French>Couleur par défaut du point de passage</French>
+                <German>Standardfarbe der Wegpunkte</German>
             </Key>
             <Key ID="STR_dui_linecompass_default_waypoint_color_desc">
                 <English>Waypoint given by squad leader or mission</English>
                 <French>Point de repère donné par le chef d'équipe ou la mission</French>
+                <German>Wegpunkt, der durch Gruppenführer oder die Mission eingestellt ist</German>
             </Key>
             <Key ID="STR_dui_linecompass_custom_waypoint_color">
                 <English>Custom Waypoint Color</English>
                 <French>Couleur personnalisée du point de passage</French>
+                <German>Personalisierte Farbe für Wegpunte</German>
             </Key>
             <Key ID="STR_dui_linecompass_custom_waypoint_color_desc">
                 <English>Waypoint created when shift+click is on the map is used</English>
-                <French>Création d'un point de passage lorsque shift+click est utilisé sur la carte</French>
+                <French>Création d'un point de passage lorque shift+click est utilisé sur la carte</French>
+                <German>Wegpunkte, die mit Shift+Klich auf der Karte erstellt wurden</German>
             </Key>
             <Key ID="STR_dui_linecompass_ace_fingering_color">
                 <English>ACE Pointing Color</English>
                 <French>Couleur du pointage ACE</French>
+                <German>Anzeigefarbe von ACE</German>
             </Key>
             <Key ID="STR_dui_linecompass_draw_bearings_none">
                 <English>None</English>
                 <French>Aucun</French>
+                <German>Keine</German>
             </Key>
             <Key ID="STR_dui_linecompass_draw_bearings_bearings">
                 <English>Quarter Bearings</English>
                 <French>Points cardinaux</French>
+                <German>Viertel Schritte</German>
```

Comment:
hier sind eigentlich die Cardinal und Ordinal richtungen gemeint. https://github.com/diwako/diwako_dui/pull/245/files hier im PR wird der englische test auch schon angepasst wo glaube ich die deutsche version auch dieses dann übersetzten sollte

Reply:
```suggestion
                <German>Haupt- und Nebenrichtungen</German>
```

We can also wait with this PR until #245 merges and I can add the translations in here of strings added there.